Traceback (most recent call last):
  File "/sandbox/code/error_tests/division_error_20250610_110632_285847_division_error.py", line 2, in <module>
    x = 1 / 0
        ~~^~~
ZeroDivisionError: division by zero
